### DISCUSSION TITLE
Interview Request

### DISCUSSION THREAD

**Discussion by ghost** [jocmp/capyreader#275] [Q&A]
Hello Josiah , I hope you had a good day. I liked what you wrote, I hope there will be more to come. I want to interview you for my personal blog. I will send the questions all at once and in bulleted form. 

Note: It will be published in Turkish. 

İsmail Şevik 
https://ismailsevik.com

**Comment by jocmp** [comment]
hey @ismailsevik - feel free to send any questions to support@asteriastudio.com and I'll be happy to take a look.
